nx/include/switch/services/fs.h

```diff
@@ -77,6 +77,19 @@ typedef struct
     u64 unk_x38;                    ///< 0 for SystemSaveData/SaveData.
 } FsSave;
 
+/// SaveCreate Struct
+typedef struct
+{
```

Comment:
Move to previous line.

Reply:
Also doc these fields. Did you verify that sizeof(FsSaveCreate) is correct?

Reply:
Update wiki too.

Reply:
Yeah the size should be 0x40 which I verified is correct. Will update the wiki shortly.

Reply:
Done.